e2e/sanity_test.go

```diff
@@ -35,47 +37,47 @@ func testSanity() {
 			g.Expect(err).ShouldNot(HaveOccurred())
 			g.Expect(ds.Status.NumberAvailable).To(BeEquivalentTo(1))
 		}).Should(Succeed())
-	})
```

Comment:
It seems that no need to create tc and thinTC in a BeforeEach block. I would like to know if there is a reason.

Reply:
It is because Address or other fields depend on nonControlPlaneNodeCount which will be initialized in BeforeSuite.

Reply:
I noted it here: https://github.com/topolvm/topolvm/pull/807/files#diff-df24cc5d6cd0a9d1708368dc0c4119fc7ac5ef7ef7293bc004d64985d736739fR21

Reply:
Thank you.